system/jlib/jsocket.cpp

```diff
@@ -4706,7 +4710,11 @@ class CSocketEpollThread: public CSocketBaseThread
                                     }
                                     if (ep_mode != 0) {
                                         tonotify.append(*epsi);
+#ifdef _TRACELINKCLOSED
+                                        tonotify.element(tonotify.length()-1).mode = epevents[j].events;
```

Comment:
minor: I know this is temporary code we intend to remove and it no existing definition of notifySelected() uses 'selected' now, but I comment explaining a) temporary and b) using it because nothing currently uses the mode passed in, so kludge for tracing would be useful here.